File: Kylie_Guzman/henry_guzman/mis_programas/dart/junio_26_2024.dart
import 'dart:io';

void main (){

  int cont = 0;
  int acum = 0;

  int cont2 =0;
  int acum2 =0;

  int cantidad = 0;
  int users = 0;
  double promedio = 0;
  double promedio2 = 0;
  double promedio3 = 0;
  
  int h = 0;

  while (h < 500.000){
    
    print ('ingrese la cantidad aportada por el aprendiz');
    int? variable = int.parse(stdin.readLineSync()!);

    if (variable >= 5000){

       cont = cont +1;
       acum = acum + variable;

    }else{

       cont2 = cont2 ++;
       acum2 = acum2 + variable;

    }
  
   cantidad = acum + acum2;
   users = cont + cont2;
   promedio = cantidad / users;
   promedio2 = acum / cont;
   promedio3 = acum2 / cont2;
  
  }

  print('la cantidad total de aportes es de $cantidad entre $users y el promedio es $promedio');
  
  print('los que aportaron 5000 0 mas = $cont');
  print('total = $acum');
  print('promedio = $promedio2');

  print('los que aportaron menos de 5000 = $cont');
  print('total = $acum2');
  print('promedio = $promedio3');

}

File: Kylie_Guzman/henry_guzman/mis_programas/ruby/junio_26_2024.rb

while h < 500.000

    puts "ingrese la cantidad aportada por el aprendiz"
    variable = gets.chomp.to_i

    if variable >= 5000

        cont = cont ++
        acum = acum + variable

    else 

        cont2 = cont2 +1
        acum2 = acum2 + variable

    end

    cantidad = h
    promedio = cantidad / h
    promedio2 = acum / cont
    promedio3 = acum2 / cont2

end

puts "la cantidad total de aportes es de #{cantidad} y el promedio es de #{promedio}"

puts "los que aportaron 5000 0 mas = #{cont}"
puts "total = #{acum}"
puts "promedio = #{promedio2}"

puts "los que aportaron menos de 5000 = #{cont2}"
puts "total = #{acum2}"
puts "promedio = #{promedio3}"

